src/components/Fullscrn_qrcode.vue

```diff
@@ -3,14 +3,17 @@
     class="mt-3"
     :class="[{ 'bubble-dark': isDarkMode }, { 'bubble-light': !isDarkMode }]"
   >
-    <h3 :class="{ 'text-white': isDarkMode }">Get our mobile app!</h3>
-
-    <div>
-      <p :class="{ 'text-white': isDarkMode }">
-        IOS APP <span :class="{ 'text-white': isDarkMode }"> Andriod APP </span>
-      </p>
+    <h3 :class="{ 'text-white': isDarkMode }">Get The Mobile App!</h3>
+    <div class="box">
+      <p :class="{ 'text-white': isDarkMode }">IOS</p>
       <div>
-        <img src="../../public/app_clip_shuttle.png" alt="ios qr code" />
+        <img src="../../public/App_Clip_Code.svg" alt="ios qr code" />
+      </div>
+    </div>
+    <div class="box">

```

Comment:
There should be more space between the bottom of the App Clip Code and the top of the "Android" text.

Reply:
fixed in 06650cb563424e310cfa3d362f0139ca70253197.